src/styles/style.css

```diff
@@ -0,0 +1,215 @@
+body {
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+  border: 2px solid red;
```

Comment:
Inline styles should not be used unless absolutely necessary. It's better to use a class to apply styles to elements. This will make the code more maintainable and scalable.